exasol_sagemaker_extension/lua/test/test_autopilot_training_main.lua

```diff
@@ -31,39 +31,33 @@ function  test_train_autopilot_main.setUp()
 end
 
 
-function test_train_autopilot_main.test_parse_unused_optional_arguments()
-    local args_empty_json_str = '{}'
-
-    args_empty = parse_arguments(args_empty_json_str)
-    luaunit.assertNil(args_empty["max_candidates"])
-    luaunit.assertNil(args_empty["problem_type"])
-    luaunit.assertNil(args_empty["objective"])
-    luaunit.assertNotNil(args_empty["compression_type"])
-end
-
 function test_train_autopilot_main.test_parse_used_optional_arguments()
-    local args_missing_json_str = '{"max_candidates" : 100, "problem_type" : "regression"}'
+    local args_missing_json_str = [[{
+       	"job_name"						    : "jobname",
+       	"aws_credentials_connection_name"	: "S3_CONNECTION",
+       	"aws_region"						: "eu-central-1",
+       	"iam_sagemaker_role"				: "arn:aws:iam::1234:role",
+	    "s3_bucket_uri"						: "s3://sme-bucket",
+	    "s3_output_path"					: "path",
+	    "input_schema_name"					: "TEST_SCHEMA",
+	    "input_table_or_view_name"			: "TABLE_NAME",
+	    "target_attribute_name"				: "TARGET",
+	    "max_candidates"					: 2
+	}]]
 
     args_missing = parse_arguments(args_missing_json_str)
     luaunit.assertNil(args_missing["max_runtime_for_automl_job_in_seconds"])
     luaunit.assertNil(args_missing["max_runtime_per_training_job_in_seconds"])
     luaunit.assertNotNil(args_missing["max_candidates"])
-    luaunit.assertNotNil(args_missing["problem_type"])
+    luaunit.assertNil(args_missing["problem_type"])
     luaunit.assertNil(args_missing["objective"])
     luaunit.assertNotNil(args_missing["compression_type"])
 end
 
-function test_train_autopilot_main.test_parse_invalid_input_string()
-    local args_invalid_json_str = '"max_candidates" : 100}'
-
-    args_invalid = parse_arguments(args_invalid_json_str)
-    luaunit.assertNil(args_invalid["max_candidates"])
-
-end
-
```

Comment:
Is there a reason, you removed this test?

Reply:
I need to capture 'exit'  function so that I can run this test.